Terminal.Gui/Application/Application.Run.cs

```diff
@@ -99,7 +99,7 @@ public static RunState Begin (Toplevel toplevel)
             else if (ApplicationOverlapped.OverlappedTop is { } && toplevel != Top && TopLevels.Contains (Top!))
             {
                 // BUGBUG: Don't call OnLeave/OnEnter directly! Set HasFocus to false and let the system handle it.
-                Top!.OnLeave (toplevel);
+                //Top!.OnLeave (toplevel);
```

Comment:
Can the line be removed?

Reply:
It def will be removed when Toplevel goes away as part of fixing #2419!

Reply:
I didn't remove it because my focus changes probably broke `OverlappedTop`. But I don't care because that's going away in the next big PR towards #2419.

Reply:
Ha understandable. Intermediate changes. 👍